app/src/main/java/com/sdpteam/connectout/event/Event.java

```diff
@@ -56,4 +56,9 @@ public String getOrganizer() {
     public List<String> getParticipants() {
         return participants;
     }
+    public void addParticipant(String id) {
```

Comment:
Neither,  the `Event` should only care about itself. Handling the add into firebase should be done through `EventFirebaseDataSource` upon joining an Event, which is my next issue. (This would be done through updating the event within firebase.)